lib/Doctrine/ruleset.xml

```diff
@@ -250,6 +250,8 @@
     </rule>
     <!-- Require language constructs without parentheses -->
     <rule ref="SlevomatCodingStandard.ControlStructures.LanguageConstructWithParentheses"/>
+    <!-- Require usage of null coalesce operator equal operator (`??`) when possible -->
```

Comment:
@carusogabriel it should be ??=

Reply:
I've removed it, just leaving the name, same as the one bellow.